frontend/src/components/App/TopBar.tsx

```diff
@@ -236,6 +252,19 @@ export function PureTopBar({
         </ListItemIcon>
         <ListItemText primary={t('Log out')} secondary={hasToken ? null : t('(No token set up)')} />
       </MenuItem>
+      <MenuItem
+        component="a"
+        onClick={() => {
+          dispatch(setVersionDialogOpen(true));
+          handleMenuClose();
+        }}
+        dense
+        className={classes.versionLink}
+      >
+        <ListItemText>
+          {window.headlampConfig.HEADLAMP_PRODUCT_NAME} {window.headlampConfig.HEADLAMP_VERSION}
```

Comment:
A cleaner way to do this would be to put this information in the store.

That would also work with a story test.